rules/performance/tests/Rector/FuncCall/PreslashSimpleFunctionRector/Fixture/skip_auto_import_names.php.inc

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Rector\Performance\Tests\Rector\FuncCall\PreslashSimpleFunctionRector\Fixture;
+
+class SkipAutoImportNames
+{
+    public function shorten($value)
+    {
+        return trim($value);
```

Comment:
If functions aren't imported, then this should be changed to `\trim` as the `PreslashSimpleFunctionRector` rule should still run normally

Reply:
fixed.